mpc/coordinator/server_configuration.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+from .crypto import \
+    VerificationKey, import_verification_key, export_verification_key
+import json
+import time
+from typing import List, Dict, cast, Optional
+
+JsonDict = Dict[str, object]
+
+TIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+class Contributor(object):
+    """
+    Details of a specific contributor
+    """
+    def __init__(self, email: str, verification_key: VerificationKey):
+        self.email = email
+        self.verification_key = verification_key
```

Comment:
I'd add an extra test before creating a new entry for a new contributor in order to make sure he/she has knowledge of the secret key associated with the registered public key. This is a way to make sure that all keys stored are "certified keys" and not just some random pk against which we will not be able to check the contributor's contribution.

Reply:
As a first pass, I've added a `make_config` tool which takes the .csv and create a kind of template configuration file so that the admin of the mpc coordinator can open the file and edit the settings for their server.

This was just the first approach I though of that was minimal effort, but more user-friendly than dumping a list somewhere (which when must be copy-pasted into the server config file).  But I'm sure there are better ways to do it (such as splitting out the contributors list and the server config file).  Let me know if you think it's worth taking a different approach.